.github/renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    ":disableDependencyDashboard",
+    ":semanticCommitsDisabled"
+  ],
+  "prHourlyLimit": 10,
+  "gomod": {}, // Upgrade go dependencies.
+  "github-actions": {}, // Upgrade GitHub actions.
+  // Renovate evaluates all packageRules and does not stop after the first match.
+  // Rules that appear later in this list override earlier rules.
+  "packageRules": [
+    // Group all Go dependencies in the same PR.
+    {
+      "matchManagers": ["gomod"],
+      "groupName": "go dependencies"
+    },
```

Comment:
This will group all go dependencies not listed below (in other groups) in the same PR. This reduces noise, but also increases friction: If a single dependency is breaking a test, it blocks the whole PR and requires us to merge manually.

Reply:
As an example, see: https://github.com/roobre/xk6-disruptor/pull/11. GRPC update breaks build and thus blocking the whole PR.

Reply:
I prefer this. We can always create a PR for updating some dependencies if we consider it is urgent and we cannot wait to fix the broken dependency.